src/features/mcda/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { toolbarControlsAtom } from '~core/shared_state';
+import { controlGroup, controlVisualGroup } from '~core/shared_state/toolbarControls';
+import { mcdaCalculationAtom } from './atoms/mcdaCalculation';
+import type { JsonMCDA } from './atoms/mcdaCalculation';
+
+const example = `
+{
+  "id": "MCDA",
+  "axes":[
+    ["population", "area_km2"],
```

Comment:
as a human, I would want to steal lines of layers using text editor (or isolate them to debug), so if the input structure can be flipped around to be an array of objects rather than an object of arrays that will help manual editing and eliminate counting errors. you can still transform it on input.

Reply:
fixed. 
json now is next: 

```
{
  "id":"MCDA",
  "layers":[
     {
        "axis":["population", "area_km2"],
        "range":[0, 46200],
        "sentiment":["good", "bad"],
        "coefficient":1
     },
     {
        "axis":["hazardous_days_count", "one"],
        "range":[0, 365],
        "sentiment":["good", "bad"],
        "coefficient":1
     },
     {
        "axis":["highway_length", "total_road_length"],
        "range":[0, 1],
        "sentiment":["bad", "good"],
        "coefficient":1
     }
  ],
  "colors":{
     "good":"rgba(90, 200, 127, 0.5)",
     "bad":"rgba(228, 26, 28, 0.5)"
  }
}
```